README.md

```diff
@@ -50,3 +66,20 @@ You should see output similar to the following:
  * Debugger PIN: 226-556-590
 ```
 Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser to view the app.
+
+## Running the Tests
+
+Install test libraries:
+
+```
+pip install -U pytest
+pip install -U mock
+pip install -U requests_mock
+pip install -U selenium
+pip install -U selenium-requests
+poetry add pytest selenium selenium-requests mock --dev
```

Comment:
If we're adding the test dependencies to poetry then that will be logged in our `pyproject.toml` and future developers won't need to worry about these steps

Reply:
Done

Reply:
Done